index.css

```
@@ -13,20 +17,18 @@ nav {
     inset: 0;
 }
 
-nav .page-log img {
-    /* width: 13.3rem;
-    height: 2.4rem; */
-    width: 133px;
-    height: 24px;    
+nav .page-logo img {
```

하위 요소들도 부모요소 셀렉터를 nav#gnb로 하는게 사이드 이펙트가 발생하지 않을 것 같아요!
(다른 코드들 전부 해당함)
```suggestion
nav#gnb .page-logo img {
```